### DISCUSSION TITLE
how to split posec3d skeleton data (in pkl file) into train/test splits?

### DISCUSSION THREAD

**Discussion by hihiok** [open-mmlab/mmaction2#2447] [Q&A]
diving48 dataset: https://download.openmmlab.com/mmaction/v1.0/skeleton/data/diving48_2d.pkl
can you provide train and test pkl files? or how to split? Thanks!

**Comment by cir7** [comment]
there is a field named 'split' in the pickle data, which provides split info. you can specify the arg `split` for PoseDataset to use separate parts for train and val stages. [here](https://github.com/open-mmlab/mmaction2/blob/main/configs/skeleton/posec3d/slowonly_r50_8xb16-u48-240e_gym-limb.py#L105) is a example
